dowhy/causal_refuter.py

```diff
@@ -38,6 +38,71 @@ def get_estimator_object(new_data, identified_estimand, estimate):
                 )
         return new_estimator
 
+    def test_significance(self, estimate, simulations):
+        """
+        Tests the satistical significance of the estimate obtained to the simulations produced by a refuter
+
+        The basis behind using the sample statistics of the refuter when we are in fact testing the estimate,
```

Comment:
I get an idea of what you are trying to convey. I'll add this to the docstring of the function. It will provide a better intuition to those who wish to understand the code better.